docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   sugar:
     build: .
     ports:
-      - "2999:2999"
+      - "2999:80"
```

Comment:
This will change the internal container port mapping to `80`, so you'll need to change the `SUGAR_PORT` the service runs on to ensure the traffic makes it through to the app as well, https://github.com/zooniverse/Sugar/blob/09b5ef54524d3f82f878f1b7ac4b8091ee153cd9/index.coffee#L4

Reply:
I'm trying to leave the container port alone, so I think I just got this backwards.